src/lib/model/ng-artifacts-factory.ts

```diff
@@ -17,7 +17,7 @@ export class NgArtifactsFactory {
     if (packageNameWithoutScope === flatModuleFileName) {
       this._makeEsmPackageNameVal = `${flatModuleFileName}.js`;
     } else {
-      const pathFromRoot = buildDirectory.replace(pathOffsetFromSourceRoot, "");
+      const pathFromRoot = buildDirectory.replace(pathOffsetFromSourceRoot, '');
```

Comment:
this is required because each package has it's own build directory, it can be removed, if everything is built in a single folder.